### PR TITLE
Bump MSRV to 1.70 following tokio 1.39

### PR DESCRIPTION
**Pull request by djc**

### CODE REVIEW

File: quinn-proto/Cargo.toml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 name = "quinn-proto"
 version = "0.11.4"
 edition = "2021"
-rust-version = "1.67"
+rust-version = "1.70"
```
Comment (mxinden) [inline quinn-proto/Cargo.toml:5]: Maybe move to workspace `Cargo.toml`?

**Review by mxinden — COMMENTED** [review]
:+1:

**Review by Ralith — APPROVED** [review]

File: quinn-proto/Cargo.toml (hunk 1 above)
Comment (Ralith) [reply quinn-proto/Cargo.toml:5]: Is that a thing? Would be neat if so.

Comment (mxinden) [reply quinn-proto/Cargo.toml:5]: Yes. See https://github.com/quinn-rs/quinn/pull/1940.

**Pull request merged**